db/schema.rb

```diff
@@ -35,8 +35,8 @@
     t.string   "type"
     t.date     "needed_date"
     t.string   "status"
-    t.datetime "created_at",         null: false
-    t.datetime "updated_at",         null: false
+    t.datetime "created_at",                     null: false
+    t.datetime "updated_at",                     null: false
```

Comment:
Yeah, this is auto-generated by rails.  I figure it is best not to hand-edit the schema.